util/io/files/tar.go

```diff
@@ -129,6 +129,11 @@ func Untgz(dstPath string, r io.Reader, maxSize int64) error {
 			}
 			f.Close()
 		}
+
+		err = os.Chtimes(target, header.AccessTime, header.ModTime)
```

Comment:
There are quite a few `header.Typeflag` values which are ignored. Can we limit this to only be called when one of the three relevant file types is specified?